app/questionnaire/router.py

```diff
@@ -63,6 +63,11 @@ def can_access_hub(self):
             if section_id in self.enabled_section_ids
         )
 
+    def can_display_summary_for_section(self, section_id, list_item_id=None):
```

Comment:
I would usually go by our tree make up, so it would be section before summary, and maybe remove the for, can_display_section_summary

Reply:
Done, thanks👍